channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/validator/JiraCloudGlobalValidator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * channel
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.channel.jira.cloud.validator;
+
+import java.util.Set;
+
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.channel.jira.cloud.descriptor.JiraCloudDescriptor;
+import com.synopsys.integration.alert.common.descriptor.config.field.errors.AlertFieldStatus;
+import com.synopsys.integration.alert.common.descriptor.validator.FieldValidator;
+import com.synopsys.integration.alert.common.descriptor.validator.GlobalValidator;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+
+@Component
+public class JiraCloudGlobalValidator extends GlobalValidator {
```

Comment:
Is Jira Cloud the POC for this feature?

Reply:
That was my intention. Seemed like a simple use case that doesn't have a million fields.